common/lib/xmodule/xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -223,6 +230,22 @@
                         $('.lang').mouseleave();
                         expect($('.lang')).not.toHaveClass('is-opened');
                     });
+
+                    it('opens the language menu on arrow up', function() {
+                        state = jasmine.initializePlayer();
+                        $('.language-menu').focus();
+                        $('.language-menu').trigger(keyPressEvent(KEY.UP));
+                        expect($('.lang')).toHaveClass('is-opened');
+                        expect($('.langs-list').find('li').last().find('.control-lang')).toBeFocused();
+                    });
+
+                    it('closes the language menu on ESC', function() {
+                        state = jasmine.initializePlayer();
+                        $('.language-menu').trigger(keyPressEvent(KEY.UP));
```

Comment:
Can you add a line here to confirm that `.lang` element has the class `is-opened`,  or maybe instead of relying on the keypress to add the class, can it be explicitly added?  Without knowing that it had class `is-opened` to start, we don't know if it was actually removed when the escape key is pressed or if the element just never had the class at all.